### DISCUSSION TITLE
ubuntu release break cypress parrellel testing.

### DISCUSSION THREAD

**Discussion by pmerwin** [actions/virtual-environments#9023] [Image configuration]
We are using cypress and cypress dashboard, we specify 22.04 for the ubuntu version in our workflows, currently our tests that run in a matrix are failing as they are getting different image versions:

So some of the matrix get: 20231211.1.0 ( chrome 119), and others get 
20231205.1.0 (chrome 120)

We would like to be able to lock an image to a version version to avoid the matrix runner's getting mixmatches.

maybe something like:
`
jobs:
  currentsTest:
    runs-on: ubuntu-22.04-20231205.1.0
    `
    so that every machine gets the same image?
    
    
This is the cypress error in the actions:

`In order to run in parallel mode each machine must send identical environment parameters such as:

specs
osName
osVersion
browserName
browserVersion (major)
This machine sent the following parameters:

{
"osName": "linux",
"osVersion": "Ubuntu - ",
"browserName": "Chrome",
"browserVersion": "120.0.6099.71.... (Expected: 1[19](https://github.com/adept-at/lib-cypress-canary/actions/runs/7197639056/job/19605228872#step:8:20))",`

**Comment by erik-bershel** [comment]
Hello @pmerwin!
This is a normal situation for the agent update process - this is a fairly long process during which more than one version of the image is available to you. Unfortunately, there is currently no way to specify the specific desired version of the image to run your tasks.

**Reply by pmerwin** [reply]
This breaks our parallel testing in prod,  anyway to make a feature request to be able to lock to a version, even if temporarily during a new image rollout, would be very beneficial.  I see I am not the first to make this request, so hopefully this gets some traction.
